packages/web-components/src/components/ic-page-header/ic-page-header.tsx

```diff
@@ -227,12 +227,14 @@ export class PageHeader {
                 {isSlotUsed(this.el, "stepper") &&
                   !isSlotUsed(this.el, "tabs") && <slot name="stepper" />}
                 {isSlotUsed(this.el, "tabs") && (
-                  <nav
-                    class="tabs-slot"
-                    aria-label="navigation-landmark-page-header"
-                  >
-                    <slot name="tabs" />
-                  </nav>
+                  <ic-horizontal-scroll>
+                    <nav
+                      class="tabs-slot"
+                      aria-label="navigation-landmark-page-header"
+                    >
+                      <slot name="tabs" />
```

Comment:
Do we need to add <ul> here to fix a11y errors with list-item role on child items? We do for top navigation etc